stubs/Factory.stub

```diff
@@ -3,10 +3,17 @@
 namespace Illuminate\Database\Eloquent\Factories;
 
 /**
- * @template TModel of \Illuminate\Database\Eloquent\Model
+ * @template TModel extends \Illuminate\Database\Eloquent\Model
```

Comment:
I have always used `extends` and am used to it from TypeScript. It seems to have worked for me, although it might have been interpreted as just a comment and never actually checked 🤷